server/neptune/workflows/internal/prrevision/workflow.go

```diff
@@ -209,3 +214,25 @@ func isRootModified(root terraform.Root, modifiedFiles []string) (bool, error) {
 
 	return false, nil
 }
+
+func (r *Runner) emitOpenPRsAgeInWeeks(ctx workflow.Context, prs []github.PullRequest, numWeeks int) {
+	ageInWeeks := make([]int, numWeeks)
+	olderThanNumWeeks := 0
+	for _, pr := range prs {
+		if age := calculateAgeInWeeks(ctx, pr); age < len(ageInWeeks) {
+			ageInWeeks[age]++
+			continue
+		}
+		olderThanNumWeeks++
```

Comment:
Again I don't think we need this, let's just have data on ageInWeeks, and then bring that stat counter in the loop + make it a gauge

Reply:
Changed it to a gauge, had to preserve the `ageInWeeks` slice since gauge can't be incremented, only set.